### ISSUE TITLE
location offset too large

### ISSUE THREAD

**Issue by middlto** [AlexeyAB/darknet#2583]
@AlexeyAB 

Hi, I am very grateful for the training problem you helped me solve a few days ago.
After 3700 iterations, I got mAP like this:
![map1](https://user-images.githubusercontent.com/22027193/54203108-808a9280-450c-11e9-952a-318702e9025a.png)

But when I test images one by one, I meet this problem:
![predictions](https://user-images.githubusercontent.com/22027193/54203701-b2e8bf80-450d-11e9-9b7a-3dd384bfb108.jpg)

Ground truth is like this:
![n03095699_138](https://user-images.githubusercontent.com/22027193/54203977-47532200-450e-11e9-99ae-27c87183c08d.JPEG)

**My question is:
Why is the location of this ship offset so much, while the width and height are accurate?** 
About 1/3 images in the training set have this problem when predicted .

I have confirmed that the coordinates of the prediction result is consistent with the location of the bounding box. So I think the problem happens in the model.

My guess:
1. Maybe I should go on training.
2. Maybe I missed some operations on image size.
3. Maybe I have a bad training set, so the model gives bad results.

Could you give me some suggestions for solving this problem? Thanks a lot!

**Comment by AlexeyAB** [comment]
@middlto Hi,

I think the issue is in the labels of Training dataset.

Check your training dataset by using Yolo_mark: https://github.com/AlexeyAB/Yolo_mark

**Comment by middlto** [comment]
@AlexeyAB 

Very very thank you for answering me! The labels do have wrong coordinates!
I made a very stupid mistake when I converted `.xml` files to` .txt` files!!!
Sorry to occupy your time on such a question!
